governance/polls/Proposal - Quantitative Easing aka Manual PSM - September 14, 2020.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Quantitative easing aka Manual PSM - September 14, 2020
+summary: Signal your support or opposition for implementing Quantitative Easing (aka Manual PSM) to fix the Peg.
+discussion_link: https://forum.makerdao.com/t/3902
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Quantitative Easing aka Manual PSM - September 14, 2020
+
+The Maker Governance Facilitator has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the Maker Governance Community. The community can use this poll to express support or opposition to a community implementation of Quantitative Easing (aka Manual PSM).
+
+This non-standard weekly Governance Poll ([FAQ](https://community-development.makerdao.com/governance/governance#is-there-more-than-one-type-of-vote)) will be active for three days beginning on Monday, September 14 at 16:00 UTC.
+
+## Review
+
+The signal request for Quantitative Easing can be found [here](https://forum.makerdao.com/t/3902) along with the results. Please review the discussion there to inform your position before voting.
+
+## Next Steps
+
+If this poll ends with a 'Yes' majority, the community will work on providing an implementation.
```

Comment:
I would maybe reword this. There is no guarantee this will happen. Maybe: "If this poll ends with a 'Yes' majority, it will signal to the community that MKR Token Holders support a community implementation."